app/scripts/minmax/dimMinMax.controller.js

```diff
@@ -325,14 +325,15 @@
         loadout.items.artifact = [items.Artifact.item];
         loadout.classType = ({ warlock: 0, titan: 1, hunter: 2 })[vm.active];
 
-        return $q.when(dimStoreService.getActiveStore())
-          .then(function(current) {
-            return dimLoadoutService.applyLoadout(current, {
-              classType: -1,
-              name: 'minmax',
-              items: items
-            }, true);
-          });
+        loadout = angular.copy(loadout);
+
+        _.each(loadout.items, function(val) {
+          val[0].equipped = true;
+        });
+
+        return $q.when(dimStoreService.getActiveStore()).then(function(current) {
```

Comment:
just c&p'd what was there, did look into that though, and was a bit confused. can update.